README.md

```diff
@@ -106,6 +106,26 @@ const output = {
 }
 ```
 
+## Usage with TypeScript
+You can use TypeScript definition from [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/inline-style-prefixer) using [@types/inline-style-prefixer](https://www.npmjs.com/package/@types/inline-style-prefixer)
+
+```sh
+npm install --save @types/inline-style-prefixer
+```
+
+Then import in your code:
+
+```typescript
+import prefixAll = require('inline-style-prefixer/static');
```

Comment:
Unfortunately yes, it is older syntax recommended for commonjs modules. Not sure why using ES2015 default import does not work with ts-loader and webpack - crashed in runtime in my env. Also is was [recommended by reviewer](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/18413#discussion_r129613410).